test/new_relic/agent/database_test.rb

```diff
@@ -453,7 +454,7 @@ def test_capture_query_long_query
     assert_equal('a' * (NewRelic::Agent::Database::MAX_QUERY_LENGTH - 3) + '...', truncated_query)
   end
 
-  INVALID_UTF8_STRING = (''.respond_to?(:force_encoding) ? "select \x80".force_encoding('UTF-8') : "select \x80")
+  INVALID_UTF8_STRING = String.new("select \x80").force_encoding('UTF-8')
```

Comment:
This string gets mutated, needs `String.new`. NOTE: we no longer test with rubies that don't respond to `#force_encoding`.